src/card.css

```
@@ -0,0 +1,106 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px var(--default-padding)
+    var(--default-padding)
+    var(--default-padding);
+  width: calc(200px - 2 * var(--default-padding) - 2 * var(--default-border));
```

you can use box-sizing: border-box 
and just set width 200px;
```suggestion
*{
  box-sizing: border-box;
}
  
```

---

agree